src/commands/apps-list.ts

```diff
@@ -0,0 +1,60 @@
+import * as clc from "cli-color";
+import * as ora from "ora";
+// TODO(caot): Replace with proper import
```

Comment:
This TODO should be addressed

Reply:
Clean up this TODO since `import .... = require(...)` is acceptable